S21/rkarmuri/ConvertToLowerCase.java

```diff
@@ -0,0 +1,98 @@
+//Method to convert given string to lowercase
+package com.company;
+
+import java.util.Locale;//Package to accept multiple languages
+
+public class ConvertToLowerCase {
+    public String convert(String wordSample, String language) {
+        System.out.println("Input word: " + wordSample);
+        System.out.println("Input language tag: " + language);
+        String wordLowerCase = wordSample.toLowerCase();//Conversion in English
+        String trazString = wordSample.toLowerCase(Locale.forLanguageTag("az"));//Conversion in Turkish and Azerbaijani
```

Comment:
Can we check the code with Checkstyle for the whitespaces errors?

Reply:
Yes you can.